CounterApp/Lock/LockCore.swift

```diff
@@ -0,0 +1,60 @@
+//
+//  LockCore.swift
+//  CounterApp
+//
+//  Created by Wenjuan Li on 2022/7/12.
+//
+
+import ComposableArchitecture
+import Foundation
+
+struct LockState: Equatable {
+    var counters: IdentifiedArrayOf<CounterState> = []
+    var isPresent: Bool = false
```

Comment:
The presentation of this as a sheet belongs actually to its parent's domain, so this is probably better declared in `AppState`.

Reply:
@nanocba Thanks! Fixed.